_episodes/035-daniel-stenberg.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Rust in cURL"
+date: 2021-09-03T23:00:00Z
```

Comment:
This is a fairly late time on Friday (https://everytimezone.com/s/06ed398f). I would probably make this an earlier time, just because podcast clients sometimes get confused for episodes with times too far into the future.